analytics-webapps/src/main/webapp/vue-app/common-components/components/samples/SampleItem.vue

```diff
@@ -45,44 +45,26 @@
     </v-expansion-panel-header>
     <v-expansion-panel-content>
       <template v-if="chartDataProps">
-        <v-row
+        <analytics-sample-item-attribute
           v-for="chartDataProp in chartDataProps"
           :key="chartDataProp"
-          class="ma-2"
-          no-gutters>
-          <v-col>{{ getI18N(chartDataProp) }}</v-col>
-          <v-col v-if="chartDataProp === 'userId' && userIdentity" class="text--secondary">
-            {{ chartData[chartDataProp] }}
-            (<analytics-profile-chip :identity="userIdentity" />)
-          </v-col>
-          <v-col v-else-if="chartDataProp === 'spaceId' && spaceIdentity" class="text--secondary">
-            {{ chartData[chartDataProp] }}
-            (<analytics-profile-chip :identity="spaceIdentity" />)
-          </v-col>
-          <v-col v-else class="text--secondary">
-            {{ chartData[chartDataProp] }}
-          </v-col>
-        </v-row>
+          :value="chartDataProp"
+          :chart-data="chartData"
+          :users="users"
```

Comment:
you may not need this anymore, since it's used for users caching. In fact, you can simply use `this.$root.users` & `this.$root.spaces`.